File: get_session.py
import asyncio
from playwright.async_api import async_playwright

async def get_claude_session():
    async with async_playwright() as p:
        browser = await p.firefox.launch(headless=False)
        context = await browser.new_context()
        page = await context.new_page()
        
        await page.goto('https://claude.ai')
        
        print("\n=== LOGIN TO CLAUDE ===")
        print("Waiting for you to login...")
        print("After login, press Enter here...")
        
        input()
        
        cookies = await context.cookies()
        
        for cookie in cookies:
            if cookie['name'] == 'sessionKey':
                print(f"\n=== SESSION KEY FOUND ===")
                print(f"\n{cookie['value']}\n")
                print("Copy this and paste to .env as CLAUDE_SESSION_KEY")
                break
        else:
            print("\nSession key not found!")
        
        await browser.close()

asyncio.run(get_claude_session())

File: get_session.js
const readline = require('readline');

console.log('\n=== CLAUDE SESSION KEY EXTRACTOR ===\n');
console.log('1. Buka Chrome di HP');
console.log('2. Login ke claude.ai');
console.log('3. Buka https://www.editthiscookie.com atau install extension EditThisCookie');
console.log('4. Atau cara manual:\n');
console.log('Jalankan ini di Termux:\n');
console.log('termux-clipboard-set "$(curl -s https://claude.ai -H "Cookie: $(termux-clipboard-get)" -v 2>&1 | grep sessionKey)"');
console.log('\n=== ALTERNATIVE: Use Groq (Free, No Session Needed) ===\n');
console.log('Visit: https://console.groq.com\n');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('Paste your session key here (or press Enter to skip): ', (answer) => {
  if (answer.trim()) {
    console.log('\n✓ Session key received!');
    console.log('Add this to your .env:');
    console.log(`CLAUDE_SESSION_KEY=${answer.trim()}`);
  } else {
    console.log('\nSkipped. Consider using Groq instead (free & legal).');
  }
  rl.close();
});
